greenmail.disable property doesn't work propely

I set grails.plugin.greenmail.disabled=true .
I'm getting NPE exception when I try to access http://localhost:8080/greenmail/list
```
Caused by: java.lang.NullPointerException: Cannot invoke method getReceivedMessages() on null object
	at com.piragua.greenmail.GreenmailController$_list_closure1$_closure4.doCall(GreenmailController.groovy:30)
	at org.grails.plugins.web.api.MimeTypesApiSupport.getResponseForFormat(MimeTypesApiSupport.groovy:142)
	at org.grails.plugins.web.api.MimeTypesApiSupport.resolveAllFormat(MimeTypesApiSupport.groovy:103)
	at org.grails.plugins.web.api.MimeTypesApiSupport.withFormatInternal(MimeTypesApiSupport.groovy:52)
	at org.grails.plugins.web.api.MimeTypesApiSupport.withFormat(MimeTypesApiSupport.groovy:44)
	at grails.artefact.Controller$Trait$Helper.withFormat(Controller.groovy:92)
	at com.piragua.greenmail.GreenmailController.list(GreenmailController.groovy:28)
	... 38 common frames omitted
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,6 @@
 language: groovy
 jdk:
-- oraclejdk7
+- openjdk8
 before_script:
 - rm -rf target
 script: ./travis-build.sh

--- a/grails-app/controllers/greenmail/GreenmailUrlMappings.groovy
+++ b/grails-app/controllers/greenmail/GreenmailUrlMappings.groovy
@@ -1,0 +1,12 @@
+package greenmail
+
+import grails.util.Holders
+
+class GreenmailUrlMappings {
+
+    static mappings = {
+        if (!Holders.grailsApplication.config.getProperty("grails.plugin.greenmail.disabled", Boolean, false)) {
+            "/greenmail/$action?/$id?(.$format)?"(controller: 'greenmail')
+        }
+    }
+}

--- a/grails-app/init/greenmail/BootStrap.groovy
+++ b/grails-app/init/greenmail/BootStrap.groovy
@@ -1,3 +1,5 @@
+package greenmail
+
 class BootStrap {
 
     def init = { servletContext ->

